### PR TITLE
ROX-18001: update timestamp for manual drools entry

### PR DESCRIPTION
**Pull request by dcaravel**
Builds on https://github.com/stackrox/scanner/pull/1205

The [previous manual](https://github.com/stackrox/scanner/blob/b75f0eb8b4be98aa394386b532a4ba178f29dafb/pkg/vulnloader/nvdloader/manual.go#L827-L830) entry [is ignored](https://github.com/stackrox/scanner/blob/24179394818ef1a9e6b0bf67d7588eb0d5fc9dd3/cmd/updater/diffdumps/cmd.go#L183-L185) by the `diff-dump` jobs because the `LastModifiedDate` of the entry occurred before the `until date` in the `genesis` dumps.  

By updating the `LastModifiedDate` the entry should now be processed by the `diff-dump` jobs.

### CODE REVIEW

File: pkg/vulnloader/nvdloader/manual.go [hunk 1]
```diff
@@ -923,7 +923,7 @@ var manuallyEnrichedVulns = map[string]*schema.NVDCVEFeedJSON10DefCVEItem{
 				ImpactScore:         5.9,
 			},
 		},
-		LastModifiedDate: "2022-06-28T13:56Z",
+		LastModifiedDate: "2023-07-06T09:30Z",
 		PublishedDate:    "2022-06-16T10:15Z",
```
Comment (RTann) [inline pkg/vulnloader/nvdloader/manual.go:927]: can you update this, as well?

**Review by RTann — APPROVED** [review]

**Comment by dcaravel** [comment]
`ci/prow/e2e-tests` failure unrelated to change

**Pull request merged**